content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -190,15 +205,15 @@ The incorrect math of the production code is trusted in every test above; all te
+Size (or scope) is one of the more useful ways to classify a test. When automating tests, prefer writing many small tests over writing fewer, larger tests. Larger tests take more initial effort to write and continuously consume magnitudes more time to execute and debug. If you can catch an issue with a smaller test, write the smaller test!
 
 ### Unit tests
 
 The most straightforward automated tests are unit tests. Unit tests have a very simple mantra: **One input, one call, one output**
 
-The purpose of a unit test is to be small, specific, and fast. They are easy to run, and when they fail, it tends to be easy to determine why. This ease of use primarily comes from the "one call" aspect. Different interactions are treated as additional tests rather than different steps within the same test with a more extended workflow. Think of a unit test as creating an extremely simple robot that quickly reports if one specific behavior breaks. Each robot allows the author to reduce expending mental resources worrying about a design concern. It also documents and monitors that concern for anyone unfamiliar with the area. This allows developers to focus on changing production code while trusting their robots will efficiently help them detect and debug issues as they get introduced. Write unit tests when writing code intended to exist for more than a week.
+The purpose of a unit test is to be small, specific, and fast. They are easy to run, and when they fail, it tends to be easy to determine why. This ease of use primarily comes from the "one call" aspect. Different interactions are treated as additional tests rather than different steps within the same test with a more extended workflow. Think of a unit test as creating an extremely simple robot that quickly reports if one specific behavior breaks. Each robot allows the author to reduce expending mental resources worrying about a design concern. It also documents and monitors that concern for anyone unfamiliar with the area. This allows developers to focus on changing production code while trusting their robots will efficiently help them detect and debug issues as they get introduced. Consider unit tests when writing code intended to exist for more than a week.
 
-A good pattern to follow when writing unit tests is ***Arrange, Act, Assert***. First, set up your environmental state (Arrange), then call the function under test (Act), and finally, verify the expected side effects occurred (Assert). This simple structure leads to short, focused tests which are easy to understand. Below is a simple example in C++:
+A good pattern to follow when writing unit tests is ***Arrange, Act, Assert***. First set up your environmental state (Arrange), then call the function under test (Act), and finally verify the expected side effects occurred (Assert). This simple structure leads to short, focused tests which are easy to understand. Below is a simple example in C++:
```

Comment:
- I think having the commas is appropriate here. The problem might be using commas to separate the steps. When listing sequential steps in one sentence, they can be joined together using semicolons instead. 
- The colon after "Arrange, Act, and Assert" simply indicates that the following expands from it.
- Don't need to italicize and bold. I think only italicizing is fitting, since that's what we use to emphasize a term or method.
```suggestion
A good pattern to follow when writing unit tests is *Arrange, Act, Assert*: first, set up your environmental state (Arrange); then, call the function that's under test (Act); finally, verify that the expected side effects occurred (Assert). This simple structure leads to short, focused tests that are easy to understand. Below is a simple example in C++:
```

Reply:
I don't see how adding the colon, semicolons, and commas help this sentence. Perhaps it sould become a bulleted list?

Bold and italics was already recommended and accepted in https://github.com/o3de/o3de.org/pull/2059

Reply:
The change I'm seeing here is to remove the commas, is that right? I think keeping the commas is appropriate. And the subsequent suggestion I make is to break apart the sentence, "First... then... finally...", so it's not one long run-on sentence.